src/routes/help/components.svelte

```diff
@@ -0,0 +1,238 @@
+<script>
+	import components from '../components/components.json';
+
+	const allTags = Array.from(new Set(components.map((item) => item.tags).flat()));
+	const tagItems = allTags.map((tag) => ({ label: tag, value: tag })).sort(ascending);
+	const allCategories = Array.from(new Set(components.map((item) => item.category).flat()));
+	const categoryItems = allCategories
+		.filter((cat) => cat !== '')
+		.map((cat) => ({ label: cat, value: cat }))
+		.sort(ascending);
+
+	let clipboardCopy = false;
+	const copyToClipboard = (text) => {
+		navigator.clipboard
+			.writeText(text)
+			.then(() => {
+				clipboardCopy = true;
+				setTimeout(() => (clipboardCopy = false), 1000);
+			})
+			.catch(() => alert('Clipboard copy Permission denied'));
+	};
```

Comment:
This function should be put inside a dedicated file as it is used in several file now

Reply:
I agree. Along with this, there are several other logic duplications (allTags, allCategories) across the project. They all should Ideally be put inside a separate file. Maybe subject for a different PR?

Reply:
I just made it: #115 😉